### ISSUE TITLE
Accessibility Issues - redundant title attributes in left nav, missing lang attribute

### ISSUE THREAD

**Issue by michaelrobertson-fico-com** [storybookjs/storybook#9202]
**Redundant Title Attributes**
the Wave Accessibility - chrome extension flags one issue, the redundancy of the title attributes on the left navigation stories (below). It's never a good idea to repeat the exact same string as the content of an element in it's title attribute.

**Example of left nav node: (mouseover to see redundant title tooltip)**
<a title="Overview" id="explorertabset--overview" class="css-1uwvbks" href="/staging-iris/?path=/story/tabset--overview"><div id="tabset--overview" kind="Tabset" class="css-4b1qgw"><span class="css-1s7dbbx"></span><svg viewBox="0 0 1024 1024" class="css-16z45m6"><path d="M772 1012L511 761l-260 251a49 49 0 0 1-52 10c-18-7-29-24-29-43V132c0-25 21-46 47-46h588c26 0 47 21 47 46v847c0 19-11 36-29 43a49 49 0 0 1-51-10zM545 664l213 205V181H265v688l213-205c9-9 21-14 33-14s24 5 34 14z" class="css-kqzqgg"></path></svg><span>Overview</span></div></a>

---------------------------------------------------
**Missing Language declaration on HTML tag**
The storybook HTML story template should include a language attribute. Ideally the language should be a variable since everyone using storybook may not be creating english sites.
`<html lang="en">` This is flagged in validators.

**Comment by michaelrobertson-fico-com** [comment]
Don't understand the message that this is inactive, both of these issues need to be resolved by the team.

**Comment by shilman** [comment]
@michaelrobertson-fico-com The message is pretty self-explanatory. If you want to take a cut at this, PRs are very welcome!
